x-pack/plugins/ingest_manager/server/saved_objects/index.ts

```diff
@@ -202,21 +202,21 @@ const savedObjectTypes: { [key: string]: SavedObjectsType } = {
         inputs: {
           type: 'nested',
           properties: {
-            type: { type: 'keyword' },
-            enabled: { type: 'boolean' },
-            processors: { type: 'keyword' },
-            config: { type: 'flattened' },
-            vars: { type: 'flattened' },
+            type: { type: 'keyword', index: false },
+            enabled: { type: 'boolean', index: false },
+            processors: { type: 'keyword', index: false },
+            config: { type: 'flattened', index: false },
+            vars: { type: 'flattened', index: false },
             streams: {
               type: 'nested',
               properties: {
-                id: { type: 'keyword' },
-                enabled: { type: 'boolean' },
-                dataset: { type: 'keyword' },
-                processors: { type: 'keyword' },
-                config: { type: 'flattened' },
-                agent_stream: { type: 'flattened' },
-                vars: { type: 'flattened' },
+                id: { type: 'keyword', index: false },
+                enabled: { type: 'boolean', index: false },
+                dataset: { type: 'keyword', index: false },
+                processors: { type: 'keyword', index: false },
+                config: { type: 'flattened', index: false },
+                agent_stream: { type: 'flattened', index: false },
```

Comment:
Note that this change by itself won't resolve #69682.

See [this comment](https://github.com/elastic/kibana/issues/69682#issuecomment-648840062) and the one below.

I got with working by also adding `doc_values: false`. But maybe you find a way you like better, like changing the type.

Reply:
I changed it to have the whole field as `enabled: false` and just tested it it seems to work